### PR TITLE
Create Poscon Banner 

### PR DESCRIPTION
**Pull request by HarryAmmon**
#### Checklist

- [ ] Includes tests
- [ ] Update documentation

### CODE REVIEW

**Comment by sussexrick** [comment]
Just one tip up-front - the banner should have a `role="alert"` attribute, eg `<div role="alert"></div>`. This means that assistive technology will draw attention to it when it appears (eg a screen reader would read it out).

**Comment by HarryAmmon** [comment]
Marking as ready for review, why is centre-ing divs such a pain?

File: packages/layout/src/components/poscon/poscon.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+	) : (
+		<PersistentPoscon color={color}>{children}</PersistentPoscon>
+	);
+};
+
+const ClosablePoscon: React.FC<ColorProps> = ({ color, children }) => {
+	const [closed, setClosed] = useState<boolean>(false);
+	return !closed ? (
+		<div role="alert">
+			<Flex cfg={{ justifyContent: 'space-between', bg: color }}>
```
Comment (sussexrick) [inline packages/layout/src/components/poscon/poscon.tsx:31]: <Flex /> generates a <div />, so you can probably just put `role=alert` onto that to save unnecessary nesting. Also the redundant `<div></div>` can go. If you're using it to create space then that should be done with CSS, eg add a class to the Flex or update `Styles.content`.

**Review by sussexrick — CHANGES_REQUESTED** [review]

**Comment by sussexrick** [comment]
Re: your comment about centring divs, if you're stuck on some CSS feel free to show me on a call and we can debug together.

File: packages/layout/src/components/buttons/buttons.tsx [hunk 2]
```diff
@@ -59,3 +60,20 @@ export const ArrowButton: React.FC<ArrowButtonProps> = ({
...
+};
+
+export const CrossButton: React.FC<CrossButtonProps> = ({
+	onClick,
+	colour,
+}) => {
+	return (
+		<div className={styles.crossButton}>
+			<Cross colour={colour} />
+			<button type={'button'} onClick={onClick}></button>
```
[the remark this answers is not in the document]
Comment (HarryAmmon) [reply packages/layout/src/components/buttons/buttons.tsx]: cross has been moved inside button to act as label

File: packages/icons/src/icons.tsx [hunk 3]
```diff
@@ -201,3 +201,38 @@ export const ArrowRight: React.FC<SVGProps> = (props) => {
...
+};
+
+export const Cross: React.FC<CrossProps> = ({ svgProps, colour }) => {
+	const strokeColour = colour
+		? colour === 'white'
+			? '#FFFFFF'
+			: '#000000'
+		: '#000000';
+	return (
+		<SVG testId="cross" {...svgProps} width="15px">
```
[the remark this answers is not in the document]
Comment (HarryAmmon) [reply packages/icons/src/icons.tsx]: `role='img` and `aria-label` added as SVG is used as label for `<CrossButton/>`

File: packages/layout/src/components/buttons/buttons.module.scss [hunk 4]
```diff
@@ -2,3 +2,18 @@
 	padding-left: 0px !important;
 	padding-right: 0px !important;
 }
+
+.crossButton {
+	display: grid;
```
[the remark this answers is not in the document]
Comment (HarryAmmon) [reply packages/layout/src/components/buttons/buttons.module.scss]: Placing the `<Cross />` component inside the `<button>` means this is no longer required

**Comment by HarryAmmon** [comment]
> Re: your comment about centring divs, if you're stuck on some CSS feel free to show me on a call and we can debug together.

A refactor has made centre-ing this easier, used Flexbox to position child items

**Review by Sitecorgi — APPROVED** [review]

**Pull request merged**